src/components/Menu/test.tsx

```diff
@@ -40,6 +40,39 @@ describe('<Menu />', () => {
     expect(fullMenuElement).toHaveStyle({ opacity: 0 })
   })
 
+  it('should handle open menu when ENTER key is pressed', () => {
+    const { container } = render(<Menu />)
+
+    // selecionar o nosso MenuFull
+    const fullMenuElement = screen.getByRole('navigation', { hidden: true })
+
+    // pressionar a tecla TAB para selecionar o menu
+    fireEvent.keyUp(container, { key: 'Tab' })
+
+    // clicar no botão de abrir o menu e verificar se ele abriu
+    fireEvent.keyUp(container, { key: 'Enter' })
```

Comment:
Another tip, instead of using `fireEvent` you can use the [`userEvent`](https://github.com/testing-library/user-event#keyboardtext-options), which is the package that they recommend to use now.

It would be like:

```
userEvent.keyboard('{Enter}')
```